docs/can-guides/commitment/recipes/todomvc-with-steal/17-routing/index.js

```diff
@@ -41,6 +40,4 @@ route("{filter}");
 route.start();
 
 var frag = view(appVM);
-document.body.appendChild(frag);
-
-require("can-todomvc-test")(appVM);
+document.body.appendChild(frag);import 'can-todomvc-test';(appVM);
```

Comment:
This `import` should be moved to the top so the function can be called correctly here. Like it's done up above in 15-setter-toggle/index.js (and some other examples probably).